Palindrome.java

```diff
@@ -0,0 +1,17 @@
+class PalindromeExample{  
+ public static void main(String args[]){  
+  int r,sum=0,temp;    
```

Comment:
Please follow the teams coding conventions, and to ensure readability please have one variable per line